hrm-domain/hrm-core/unit-tests/case/caseService.test.ts

```diff
@@ -272,6 +273,7 @@ describe('searchCases', () => {
           user: twilioUser(workerSid, []),
           permissions: rulesMap.open,
         },
```

Comment:
An extra test or two verifying the behaviour for both flag settings might be nice, not blocking tho

Reply:
I've actually got an edge case bug while writing the unit tests xD